server/src/webserver.rs

```diff
@@ -48,6 +48,16 @@ use urlencoding::{self, encode};
 #[cfg(target_os = "windows")]
 use win_partitions::win_api::get_logical_drive;
 
+use html5ever::tendril::TendrilSink;
```

Comment:
Run `cargo fmt` (and also `cargo clippy`) to clean up imports / check for lint.

Reply:
Done.  This also cleaned up the tabs and spacing.  Please see updated code.